### PR TITLE
WIP: Fix accessiblity in tooltip and popover

### PR DESCRIPTION
**Pull request by jaortiz**

### CODE REVIEW

File: components/tooltip/src/Tooltip.js [hunk 1]
```diff
@@ -97,7 +99,7 @@ Tooltip.propTypes = {
 	/**
 	 * Tooltip tag
 	 */
-	tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+	// tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
```
Comment (jonnystening) [inline components/tooltip/src/Tooltip.js]: Wanna remove this?

**Comment by jonnystening** [comment]
Sorry @jaortiz ... it also needs to programmatically set focus on the bubble when it opens

**Comment by jaortiz** [comment]
@jonnystening  Do we want to focus trap the tooltip/popover? (like in modal?)

**Comment by jonnystening** [comment]
Only need to focus trap when we’re isolating the tabbable items on a UI layer. Usually only when a dark overlay background Is shown... modal and off-canvas navs

**Comment by jonnystening** [comment]
> Only need to focus trap when we’re isolating the tabbable items on a UI layer. Usually only when a dark overlay background Is shown... modal and off-canvas navs

Sorry, ignore this. Yes we need to focus trap. Just in a kind of different way to how its done in Modal

**Pull request closed without merging**